agent-tooling/src/test/groovy/io/opentelemetry/auto/test/HelperInjectionTest.groovy

```diff
@@ -33,13 +32,15 @@ import java.util.concurrent.atomic.AtomicReference
 import static io.opentelemetry.auto.test.utils.ClasspathUtils.isClassLoaded
 import static io.opentelemetry.auto.tooling.ClassLoaderMatcher.BOOTSTRAP_CLASSLOADER
 import static io.opentelemetry.auto.util.gc.GCUtils.awaitGC
+import static org.junit.Assume.assumeTrue
 
 class HelperInjectionTest extends AgentSpecification {
 
   @Timeout(10)
   def "helpers injected to non-delegating classloader"() {
     setup:
     String helperClassName = HelperInjectionTest.getPackage().getName() + '.HelperClass'
+    assumeTrue(!isClassLoaded(helperClassName, Utils.getAgentClassLoader()))
```

Comment:
@trask Yup! My biggest concern right now is this test, not sure why it's not passing with JUnit5. The issues with assumeTrue I'll need to file an issue with JUnit team for.